angularjs-portal-home/src/main/webapp/partials/layout.html

```diff
@@ -38,11 +38,11 @@ <h3 class="home-section-header portlet-section-header">
        <li class="col-xs-12 col-sm-6 col-md-6 col-lg-4 no-padding portlet-container-home" ng-repeat="portlet in layout">
           <default-card 
             ng-if="!$storage.staticContentOnHome || 
-                    portlet.staticContent == null">
+                    portlet.pithyStaticContent == null">
           </default-card>
           <static-content-card 
             ng-if="$storage.staticContentOnHome &&
-                    portlet.staticContent != null">
+                    portlet.pithyStaticContent != null">
```

Comment:
It was at one point, haven't reviewed it in a while since its been commented out.